library/src/main/java/hotchemi/android/rate/Utils.java

```diff
@@ -14,12 +14,12 @@ static boolean underHoneyComb() {
         return Build.VERSION.SDK_INT < Build.VERSION_CODES.HONEYCOMB;
     }
 
-    static boolean isLollipop() {
-        return Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP || Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP_MR1;
+    static boolean greaterThanKitKat() {
```

Comment:
Like said in the message this was a quick solution to a bigger problem. 

Having to check if the version is Lollipop isn't enough in this case because the same condition should apply for when it is Marshmallow. Therefore I changed it to greaterThanKitKat as it wasn't working correctly in Marshmallow. 

The buttons were showing in a vertical order and that was not nice.

Changing this makes it work on Lollipop as well as Marshmallow. But like I said I think the optimal solutions is to allow the user to set its own theme in the same manner he is allowed to set his own view.